### PR TITLE
[WIP] 1.1.2 release

### PR DESCRIPTION
**Pull request by faymontage**

### CODE REVIEW

File: Token/Controllers/TabBarController.swift [hunk 1]
```diff
@@ -109,14 +109,14 @@ open class TabBarController: UITabBarController {
     
     private func presentAddressChangeAlertIfNeeded() {
         guard UserDefaults.standard.bool(forKey: AddressChangeAlertShown) == false else { return }
-        
-        DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) {
+
+        DispatchQueue.main.asyncAfter(deadline: .now() + 1.0) {
```
Comment (faymontage) [inline Token/Controllers/TabBarController.swift]: This is still a kludge, but it worked on all my test hardware at least - 0.5 failed a lot

File: Token/Controllers/Sign In/AddressChangeAlertController.swift [hunk 2]
```diff
@@ -0,0 +1,72 @@
+//
+//  AddressChangeAlertController.swift
```
Comment (yuliaveres) [inline Token/Controllers/Sign In/AddressChangeAlertController.swift]: forgot to remove

File: Token/Controllers/Sign In/AddressChangeAlertController.swift [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+//  Copyright © 2017 Bakken&B√¶ck. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import SweetUIKit
+
+final class AddressChangeAlertController: AlertController {
+    
+    lazy var textView: UITextView = {
```
Comment (yuliaveres) [inline Token/Controllers/Sign In/AddressChangeAlertController.swift]: private ( everth below as well )

File: Token/Controllers/TabBarController.swift [hunk 4]
```diff
@@ -19,6 +19,10 @@ import CameraScanner
 
 let TabBarItemTitleOffset: CGFloat = -3.0
 
+let AddressChangeAlertShown = "AddressChangeAlertShown"
+
+let alertText = "We have made changes to make\nToken compatible with other\nservices - as a result your\nbalance will appear to be reset.\nAs this is only testnet ETH,\n you don't need to take any action.\nIf you do want to recover your\nbalance for any reason, you can\nread how to do it here: http://developers.tokenbrowser.com/"
```
Comment (yuliaveres) [inline Token/Controllers/TabBarController.swift]: put inside AddressChangeAlertController

File: Token/Supporting Files/AppDelegate.m [hunk 5]
```diff
@@ -108,6 +108,9 @@ - (void)userDidSignOut {
 
 - (void)createNewUser {
     if (TokenUser.current == nil) {
+        [[NSUserDefaults standardUserDefaults] setBool:YES forKey:@"AddressChangeAlertShown"]; //suppress alert for users created >=v1.1.2
```
Comment (yuliaveres) [inline Token/Supporting Files/AppDelegate.m]: we have constant for this

**Pull request merged**